util/src/test/scala/org/broadinstitute/dsde/rawls/util/RetrySpec.scala

```diff
@@ -0,0 +1,179 @@
+package org.broadinstitute.dsde.rawls.util
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import com.typesafe.scalalogging.{LazyLogging, Logger}
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers._
+import org.mockito.Mockito._
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.mock.MockitoSugar
+import org.scalatest.time.{Minutes, Seconds, Span}
+import org.scalatest.{BeforeAndAfterAll, FlatSpecLike, Matchers}
+import org.slf4j.{Logger => Underlying}
+
+import scala.collection.JavaConverters._
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.reflect._
+
+/**
+  * Created by rtitle on 5/16/17.
+  */
+class RetrySpec extends TestKit(ActorSystem("MySpec")) with FlatSpecLike with BeforeAndAfterAll with Matchers with MockitoSugar with ScalaFutures {
+  import system.dispatcher
+
+  implicit override val patienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)))
```

Comment:
Actually not Mockito, it's ScalaTest Futures:
http://www.artima.com/docs-scalatest-2.0.M5/org/scalatest/concurrent/Futures.html

Basically it configures how long the calls to `whenReady` will wait. I'll comment it.